rotkehlchen/chain/ethereum/modules/yearn/vaults.py

```diff
@@ -373,7 +177,7 @@ def get_usd_price_zero_if_error(
     """
     inquirer = Inquirer()
     if (
-        asset in inquirer.special_tokens or
+        asset.identifier in inquirer.special_tokens or
```

Comment:
This is not a bad change since it's explicit but you should know that it was not needed here.

The `in` check for equality works due to the overloading of the operator check we have in asset, that boils it down to identifier string comparison.